gamercade_console/src/console/contexts/draw_context.rs

```diff
@@ -412,4 +425,40 @@ impl DrawContext {
             .take(pixel_count)
             .for_each(|pixel| pixel.copy_from_slice(&color));
     }
+
+    fn draw_circle(&mut self, x0: usize, y0: usize, x: usize, y: usize, color: Color) {
```

Comment:
I think this function should be renamed, since it's only drawing potentially 8 pixels around the circle and is called repeatedly. Maybe something like `draw_circle_points` would be more descriptive.